link-grammar/tokenize/tokenize.c

```diff
@@ -1062,6 +1062,17 @@ static void altfree(const char **alts)
 	free(alts);
 }
 
+static void gwappend(Word * w, Gword * gw)
```

Comment:
I think `gwordlist_append()` can be used instead.
This has the benefit of sharing code.
In this particular case, I would like to find a way to reduce the number of `malloc()` calls and change the function accordingly. (The same for `altappend()`.)

Reply:
Remarks:
* I like to pick "funny" names like `gwappend` instead of `gwordlist_append` because I study code using `grep`. When everything is called "gwordlist", it becomes very hard to find greps that return only those parts that are relevant. In this case, it probably doesn't matter. But still ... easy-to-search names are important.

* I looked at `altappend` and it is used in a lot of different places. At first, I thought I would be able to replace `altappend` by gwappend, everywhere, but this seems impossible.

Reply:
Oh, I see ... there already is a function called `gwordlist_append` .. I misunderstood. Fixing now.